moshi/src/main/resources/META-INF/proguard/moshi.pro

```diff
@@ -0,0 +1,18 @@
+# JSR 305 annotations are for embedding nullability information.
+-dontwarn javax.annotation.**
+
+-keepclasseswithmembers class * {
+    @com.squareup.moshi.* <methods>;
+}
+
+-keep @com.squareup.moshi.JsonQualifier interface *
+
+# The name of @JsonClass types is used to look up the generated adapter
+-keepnames @com.squareup.moshi.JsonClass class *
+
+# Retain generated JsonAdapters if annotated type is retained
```

Comment:
nit: Add a period after line 10 and 13? Line 1's comment has a period.

Reply:
done 24e0777